Orange/data/table.py

```diff
@@ -897,7 +897,33 @@ def __repr__(self):
 
     @classmethod
     def concatenate(cls, tables, axis=0):
-        """Concatenate tables into a new table"""
+        """
+        Concatenate tables into a new table, either horizontally or vertically.
```

Comment:
Maybe change the order here to "vertically or horizontally" to have the default mentioned first and match the order bellow.